Core/include/Acts/Seeding/EstimateTrackParamsFromSeed.hpp

```diff
@@ -295,6 +295,13 @@ std::optional<BoundVector> estimateTrackParamsFromSeed(
     params[eBoundTime] = spGlobalPositions[0].norm() / v;
   }
 
+  if (params.hasNaN()) {
+    ACTS_ERROR(
+        "The NaN value exists at the estimated track parameters from seed with"
+        << "\nbottom sp: " << spGlobalPositions[0] << "\nmiddle sp: "
+        << spGlobalPositions[1] << "\ntop sp: " << spGlobalPositions[2]);
+    return std::nullopt;
```

Comment:
Do we understand why this appears? Are the differences between seed points too small here?

Reply:
It actually happens only if seed contains two same space points, but I think it doesn't hurt to have the check here just in case some stupid mistake is made.

Reply:
I see, weird though that two space points can end up in a seed, but that's not your fault, so let's leave that check in.